content/en/tools/digital-currencies/redeeming/approving-or-rejecting-redemption-requests.md

```diff
@@ -0,0 +1,152 @@
+   * **Amount:** The number of tokens to be redeemed.
+   * **Requester:** The entity making the redemption request.
+   * **Last Updated:** The date and time at which the redemption request was last updated.
+   
+2. Click the relevant redemption request.
+
+   The **Redemption Request** dialog box is displayed:
+   
+   {{< 
+      figure
+	  src="images/approving-redemption-request.png"
+      width=40%
+	  figcaption="Redemption Request Confirmation Dialog"
+	  alt="Redemption Request Confirmation Dialog"
+   >}}  
+   
+   If the dialog box looks like the following image, that means the current participant is a custodian but the issuer has not yet approved the redemption request:
+   
+   {{< 
+      figure
+	  src="images/approving-redemption-request-issuer-not-approved.png"
+      width=40%
+	  figcaption="Redemption Request Confirmation Dialog - Not Approved by Issuer"
+	  alt="Redemption Request Confirmation Dialog - Not Approved by Issuer"
+   >}}
+   
+7. Click **Approve**.
+
+   The message **Successfully submitted your redemption approval request update** is displayed:
+   
+   {{< 
+      figure
+	  src="images/successfully-submitted-redemption-approval-request-update-message.png"
+      width=40%
+	  figcaption="'Successfully submitted your redemption approval request update' Message"
+	  alt="'Successfully submitted your redemption approval request update' Message"
+   >}}
+   
+   The **Update Approval Request** flow begins and its progress can be checked in the pull-out flow tracker on the right-hand side of the screen: 
+   
+   {{< 
+      figure
+	  src="images/update-approval-request-flow.png"
+      width=60%
+	  figcaption="Update Approval Request Flow"
+	  alt="Update Approval Request Flow"
+   >}}
+   
+   Once the **Update Approval Request** flow finishes, the message **Your request update has been successful** is displayed:
+
+   {{< 
+      figure
+	  src="images/request-update-successful-message.png"
+      width=40%
+	  figcaption="'Your request update has been successful' Message"
+	  alt="'Your request update has been successful' Message"
+   >}}
+  
+   If you are an issuer, you can now see that, in the **Redemption Requests** panel, the **Issuer Status** of the redemption request is now **Approved**:  
+  
+   {{< 
+      figure
+	  src="images/redemption-requests-panel-issuer-approved.png"
+      width=100%
+	  figcaption="Redemption Requests Panel - Approved by Issuer"
+	  alt="Redemption Requests Panel - Approved by Issuer"
+   >}}
+   
+   If you are a custodian, you can now see that, in the **Redemption Requests** panel, the **Custodian Status** of the redemption request is now **Approved**:  
+
+   {{< 
+      figure
+	  src="images/issuance-requests-panel-custodian-approved.png"
+      width=100%
+	  figcaption="Redemption Requests Panel - Approved by Issuer"
+	  alt="Redemption Requests Panel - Approved by Issuer"
```

Comment:
```suggestion
	  figcaption="Redemption Requests Panel - Approved by Custodian"
	  alt="Redemption Requests Panel - Approved by Custodian"
```
Is that correct? Wasn't sure about this one but wanted to double-check with you.